test/e2e/parallel/registration_service_test.go

```diff
@@ -613,7 +615,16 @@ func TestActivationCodeVerification(t *testing.T) {
 			states.SetApprovedManually(us, true)
 		})
 		require.NoError(t, err)
-		t.Logf("user signup '%s' reactivated", userSignup.Name)
+		t.Logf("user signup '%s' approved", userSignup.Name)
+		// check that the MUR and Space are configured as expected
```

Comment:
I only see the MUR check. Are we missing the Space check below?

Reply:
good point, let me add that!

Reply:
done in 282aa47